cou/commands.py

```diff
@@ -118,7 +118,7 @@ def purge_before_arg(value: str) -> str:
     :rtype: str
     :raises argparse.ArgumentTypeError: if string format is invalid
     """
-    formats = ["%Y-%m-%d %H:%M:%S", "%Y-%m-%d %H:%M", "%Y-%m-%d"]
+    formats = ["%Y-%m-%d%H:%M:%S", "%Y-%m-%d%H:%M", "%Y-%m-%d"]
```

Comment:
why remove the white space?

Reply:
Because without the whitespace is how it's documented:

> Date string format should be YYYY-MM-DD[HH:mm][:ss].

And it works better without whitespace for passing the value on the command line, otherwise you would need to quote it:

```
--purge-before-date "2024-01-01 01:01"
vs
--purge-before-date 2024-01-0101:01
```

It's more readable with the space, or even with the standard `T` separator (`2024-01-01T01:01`), but we have it documented without the space.

Reply:
CC @chanchiwai-ray @jneo8 @gabrielcocenza @rgildein it's probably not too late to change the format though - how would we prefer the datestamp here?

- "2024-01-01 01:01"
- 2024-01-0101:01
- 2024-01-01T01:01
- something else?

Reply:
> Because without the whitespace is how it's documented:

Thanks, I assumed that was followed from the doc somewhere

 `"2024-01-01 01:01"`  is more natural to me

Reply:
I guess for now, to avoid expanding the scope of the PR too much, the question is: do we fix the code to match the docs, or fix the docs to match the code?

Reply:
my guess it was just a typo in the doc, @jneo8 can confirm this

Reply:
Why do you want to do `--purge-before-date 2024-01-0101:01`? This is harder to understand.
I think that this will be mainly use with date, so I do not see any issue here.

Reply:
> Why do you want to do --purge-before-date 2024-01-0101:01? 

I don't, but all the COU docs say that's the format it uses.

Reply:
Ok, that seems like the consensus, thanks. I'll update the format to `YYYY-MM-DD[ HH:mm[:ss]]` to match the code. :+1: